integrations/spring-boot/pom.xml

```diff
@@ -84,28 +84,25 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
-                <configuration>
-                    <skip>true</skip>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.10.1</version>
-                <configuration>
-                    <compilerId>javac</compilerId>
-                </configuration>
                 <executions>
                     <execution>
-                        <id>process-annotations</id>
-                        <phase>generate-sources</phase>
+                        <id>enforce-files-exist</id>
                         <goals>
-                            <goal>compile</goal>
+                            <goal>enforce</goal>
                         </goals>
+                        <phase>package</phase>
                         <configuration>
-                            <compilerArgs>
-                                <arg>-proc:only</arg>
-                            </compilerArgs>
+                            <rules>
+                                <requireFilesExist>
+                                    <message>The module-info is not present</message>
```

Comment:
Message should refer to tested file

Reply:
Ok, copy paste error.

Reply:
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/17253582/188563696-b22a27bd-cb31-4a83-a774-6aa9d9842e27.png">